step-release-vis/src/app/components/tooltip/tooltip.html

```diff
@@ -1 +1,3 @@
-<!-- TODO(ancar): add HTML div element -->
+<div>
+  <p> Current candidate: {{candidateName}} </p>
```

Comment:
pls link the candidate to http://rapid/<candidatename>
no need to write current candidate before, it is obvious from the name, unless you put all the info from the snapshot into the tooltip, but even in that case making the candidate link bold would be better... 

IMO this sohuld look like
{{time of hoverd snapshot in local time RFC3339 display (YYYY.MM.DD. hh:mm[:ss] TZ}}
<a href=rapid/{{candidatename}} >{{candidatename}}:}</a> {{jobcount}}
<a href=rapid/{{candidatename}} >{{candidatename}}:}</a> {{jobcount}} << make this bold if this was the one hovered
<a href=rapid/{{candidatename}} >{{candidatename}}:}</a> {{jobcount}}
<a href=rapid/{{candidatename}} >{{candidatename}}:}</a> {{jobcount}}

Reply:
lol, the comment is actually interpreting the links, sorry, pls check the source of it.

Reply:
yes, this is only the skeleton, i wanted to add all the data step by step as i need many functions for all of them. Will add the rest in future PRs